src/cycax/__init__.py

```diff
@@ -1,8 +1,11 @@
 # SPDX-FileCopyrightText: 2023-present Martin Slabber <martin@tsolo.io>
 
 from cycax.cycad.assembly import Assembly  # noqa: F401
+from cycax.cycad.assembly_blender import AssemblyBlender  # noqa: F401
```

Comment:
Why are we adding the imports here? I do not think that is needed. Calling the assemblers is hidden behind the Assembly.render() 
Removing these imports and the ones in src/cycax/cycad/__init__.py makes the tests pass.  We need to fix test or imports.